src/main/java/me/shikichee/AnchorApplication.java

```diff
@@ -7,6 +7,11 @@
 public class AnchorApplication {
 
     public static void main(String[] args) {
+        String test = null;
+        if(test!=null){
+            System.out.println("test");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.yo1000.com/coding_rules#rule_key=squid%3AS106)